dashboard/test/models/script_test.rb

```diff
@@ -2375,25 +1971,6 @@ def populate_cache_and_disconnect_db
     )
   end
 
-  test 'pilot units are always hidden during seed' do
```

Comment:
Can you point me to the matching test for this?

Reply:
this seem worth testing. I'll add a new test case for this.

Reply:
hmm, now i am getting an error seeding csa2-pilot-facilitator. I will take another crack at this

Reply:
Let me know when you want me to take another look.

Reply:
Sorry to leave you hanging, Dani. This is now ready for another look. I changed this to use an after_save hook as explained in the added comments. I verified this as follows:
* `rake seed:scripts` --> all passing
* remove all csa?-pilot-facilitator units, and the csa-pilot-facilitator unit group, and then reseed all csa?-pilot-facilitator units. this exercises the codepath which got me earlier, where the unit is going to have a unit group, but does not yet have one because units are seeded before unit groups.